pkg/proto/stream/stream_test.go

```diff
@@ -0,0 +1,286 @@
+package stream
+
+import (
+	"bytes"
+	"fmt"
+	"testing"
+
+	"github.com/golang/protobuf/proto"
+	"github.com/stretchr/testify/require"
+	"gotest.tools/assert"
+)
+
+const fieldDub int = 1
+const fieldFlo int = 2
+const fieldI32 = 3
+const fieldI64 = 4
+const fieldU32 = 5
+const fieldU64 = 6
+const fieldS32 = 7
+const fieldS64 = 8
+const fieldF32 = 9
+const fieldF64 = 10
+const fieldSF32 = 11
+const fieldSF64 = 12
+const fieldBoo = 13
+const fieldStr = 14
+const fieldByt = 15
+const fieldEnu = 16
```

Comment:
(admittedly these are for tests, but I'd like to set a good example)

Reply:
> I don't think that'd be a good idea, since these correspond to the values in the .proto file, which are specified explicitly

I see your point though some kind of a comment to indicate this would be nice on top of the `const` stack.